### ISSUE TITLE
a few problems in command.js

### ISSUE THREAD

**Issue by armoredpigeon** [BenZed/after-effects#19]
I found (and feel free to correct me if I'm wrong) that the codify_includes() function in command.js is joining the array of includes on a comma, which is throwing an error in AfterEffects when it tries to run the resulting jsx file.

On line 109, if you just call `.join("\n");` after 
`}).filter(inc => is(inc,String))`
it will resolve the issue where After Effects complains "Unable to execute script at line ##. , is undefined".

**Comment by armoredpigeon** [comment]
Also, the line `this.includes = [...codify_includes(this.options.includes, this.options.minify), babelIncludes];` seems to break because it doesn't recognize the ... as anything meaningful. Having removed that and resolved the above issue, it seems to be working nicely.

**Comment by BenZed** [comment]
The ... is the spread operator.

I first wrote this package before regularly using a transpiler, ironically only using transpilation for the functions going into after effects.

So if you're on Node 4.7.2 or earlier, it might not recognize it.

I've got a refactor on the go (albeit slowely) that is transpiled, and will work on every Node version.

I'll leave this issue open until that version is finished or until I do a quick hotfix or something. I'll take a closer look when I'm back at the office tomorrow.

Thanks for letting me know!
